src/rust/ide/lib/parser/tests/doc-gen.rs

```diff
@@ -0,0 +1,34 @@
+use enso_prelude::*;
+
+use parser::DocParser;
+
+use uuid::Uuid;
+use wasm_bindgen_test::wasm_bindgen_test_configure;
+use wasm_bindgen_test::wasm_bindgen_test;
+
+
+
+wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
+fn doc_gen_test() {
+    // no doc case
+    let input = String::from("type Foo\n  type Bar");
+    let program = std::env::args().nth(1).unwrap_or(input);
+    let parser = DocParser::new_or_panic();
+    let gen_code = parser.doc_parser_generate_html_source(program).unwrap();
+    assert_eq!(gen_code, "{\"SuccessDoc\":{\"code\":\"\"}}");
+
+    // simple cases
+    let input = String::from("##\n  foo\ntype Foo\n");
+    let program = std::env::args().nth(1).unwrap_or(input);
+    let parser = DocParser::new_or_panic();
+    let gen_code = parser.doc_parser_generate_html_source(program).unwrap();
```

Comment:
* deserialize into ResponseDoc
* don't check html so closely (it is an implementation detail of parser, not integration responsibility) — it should be enough to know that some non-empty html is returned